rest/model/subscriber.go

```diff
@@ -97,6 +113,19 @@ func (s *APISubscriber) ToService() (interface{}, error) {
 			return nil, err
 		}
 
+	case event.GithubMergeSubscriberType:
+		apiModel := APIGithubMergeSubscriber{}
+		if err = mapstructure.Decode(s.Target, &apiModel); err != nil {
```

Comment:
`grip.Error` the error message so that if someone reports a "Subscriber target is malformed" error message, we can check our logs for details.

Reply:
Done.